lib/mongoid/association/proxy.rb

```diff
@@ -169,6 +169,17 @@ def execute_callback(callback, doc)
         end
       end
 
+      # Execute the before and after callbacks for the given method.
+      # Returns the result of the given block
```

Comment:
Usually we use `@return` tag to specify what is returned from a method. Maybe we should do same here to be consistent.

Reply:
good catch, done!